### PR TITLE
Accounting base64 encoding for resources size validation

### PR DESCRIPTION
**Pull request by rishigupta1599**
Resources which were greater than 25 MB in size we want to skip them in resource array as they might fail to upload, but when we upload we do base64 encoding which increases the size but while validation we were not accounting for increase in size due to encoding.
Base64 has 33-37% increase in content size so we are reducing max size check by that much factor

### CODE REVIEW

**Review by Amit3200 — APPROVED** [review]

File: packages/client/src/client.js [hunk 1]
```diff
@@ -342,11 +342,8 @@ export class PercyClient {
     validateId('build', buildId);
     if (filepath) {
       content = await fs.promises.readFile(filepath);
-      if (process.env.PERCY_GZIP) {
-        content = Pako.gzip(content);
-      }
     }
-    let encodedContent = base64encode(content);
+    let encodedContent = getEncodedContent(content);
```
Comment (ninadbstack) [inline packages/client/src/client.js:346]: There is already percy gzip check inside discovery. So this will cause double gzip

Comment (rishigupta1599) [reply packages/client/src/client.js:346]: In network file we are not setting it to encoded content we are just using it for size calculation

Comment (rishigupta1599) [reply packages/client/src/client.js:346]: client code is existing one just moved it to method

Comment (rishigupta1599) [reply packages/client/src/client.js:346]: Should I remove then?

Comment (ninadbstack) [reply packages/client/src/client.js:346]: No its not :P you missed a condition

**Review by ninadbstack — APPROVED** [review]

**Review by Amit3200 — APPROVED** [review]

**Pull request merged**